percy/providers/appAutomateProvider.js

```diff
@@ -84,19 +87,21 @@ class AppAutomateProvider extends GenericProvider {
         return result;
       } catch (e) {
         log.debug(`[${name}] Could not mark App Automate session as percy`);
+        log.debug(`[${name}] ${e}`);
         return null;
       }
     });
   }
 
-  async percyScreenshotEnd(name, percyScreenshotUrl, statusMessage = null) {
+  async percyScreenshotEnd(name, percyScreenshotUrl, sync, statusMessage = null) {
     return await TimeIt.run('percyScreenshotEnd', async () => {
       try {
         await this.browserstackExecutor('percyScreenshot', {
           name,
           percyScreenshotUrl,
           status: percyScreenshotUrl ? 'success' : 'failure',
           statusMessage,
+          sync,
```

Comment:
Reason to add here?

Reply:
this is an end call. There is a pr in selenium hub. This is for instrumentation.